pkg/porter/editor.go

```diff
@@ -0,0 +1,86 @@
+package porter
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+
+	"get.porter.sh/porter/pkg/context"
+)
+
+const (
+	defaultEditor        = "vi"
+	defaultEditorWindows = "notepad"
```

Comment:
Here's another way to do this that doesn't require a runtime check.

I use build tags at the top of the file to make sure that binaries built for windows get the right constant and binaries for other OS's (I'm calling that *nix) get the other definition of the same constant.

https://github.com/deislabs/porter/blob/041c93a33f0bd3eeb1b203f950bb8feb6a481ce7/pkg/pkgmgmt/pkgmgmt_windows.go

https://github.com/deislabs/porter/blob/041c93a33f0bd3eeb1b203f950bb8feb6a481ce7/pkg/pkgmgmt/pkgmgmt_nix.go

Then in the rest of your code, you can reference `defaultEditor` and not need to check at runtime which OS you are on.

Reply:
Good call, that would make this a bit cleaner. Will fix this up. Thanks!